src/types.ts

```diff
@@ -24,10 +24,12 @@ export type File = {
   path: string
 }
 
+type EngineType = 'library' | 'binary' | 'dataproxy'
```

Comment:
This belongs in https://github.com/prisma/templates-node/blob/main/src/data/prisma.ts